build.sbt

```diff
@@ -60,8 +60,8 @@ libraryDependencies ++= Seq(
   cache,
   ws,
   "org.scalaz" %% "scalaz-core" % "7.2.7",
-  "com.gu" %% "membership-common" % "0.553",
```

Comment:
official version number tbd, from this pr https://github.com/guardian/membership-common/pull/608